### ISSUE TITLE
Default code is not changing  when language is changed

### ISSUE THREAD

**Issue by alrouxnof** [Eessh/code-n-run#1]
@Eessh  When the language is changed, the default c++ code is not changing.

![Screenshot (13)](https://user-images.githubusercontent.com/62495956/208483715-1c06873b-8b45-4d05-87b6-2ea4045571e3.png)

**Comment by Eessh** [comment]
Yeah, I was thinking of assigning default codes for each language.
So when the language is changed, the editor code is changed to the default code of the active language.

**Comment by alrouxnof** [comment]
Is the issue fixed?
